code/addons/vitest/src/node/vitest-manager.ts

```diff
@@ -165,11 +164,15 @@ export class VitestManager {
   }
 
   private updateLastChanged(filepath: string) {
```

Comment:
this whole function can be replaced with `invalidateFile` which is available from Vitest 3 and was added specifically for Storybook